zebrad/tests/common/lightwalletd.rs

```diff
@@ -306,7 +306,17 @@ impl LightwalletdTestType {
 
     /// Returns the lightwalletd state path for this test, if set.
     pub fn lightwalletd_state_path(&self) -> Option<PathBuf> {
-        env::var_os(LIGHTWALLETD_DATA_DIR_VAR).map(Into::into)
+        match env::var_os(LIGHTWALLETD_DATA_DIR_VAR) {
+            Some(path) => Some(path.into()),
+            None => {
+                tracing::info!(
+                    "skipped {self:?} lightwalletd test, \
```

Comment:
Blocker: These logs are really confusing, because they don't contain the test name.

Can we also pass a string containing the test name, and log it in the skip log message?

We also call this method from the 3 `lightwalletd_integration_test` tests, so we need to make some changes there as well.

Reply:
You are right, i made some improvements in https://github.com/ZcashFoundation/zebra/pull/4264/commits/aaecc61709f720a3bb0f8c3f4460d97c72923590